include/freerdp/settings.h

```diff
@@ -1559,6 +1559,9 @@ struct rdp_settings
 	                                   default value - currently UNUSED! */
 	ALIGN64 char* ActionScript;
 	ALIGN64 DWORD Floatbar;
+
+
+	ALIGN64 BOOL UseExternalIO;
```

Comment:
Remove this, use a public function that allows setting `Read` and `Write` functions for the transport